volatility/framework/plugins/windows/vaddump.py

```diff
@@ -36,14 +36,33 @@ def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]
                 requirements.PluginRequirement(name = 'vadinfo', plugin = vadinfo.VadInfo, version = (1, 0, 0)),
                 ]
 
+    @staticmethod
```

Comment:
One convention is that we tend to use `classmethod` rather than `staticmethod`, but I can't think of anything from the class that such an extensible method (not this one in particular, but any in the future) might want to use?  I have the nagging feeling I did try `staticmethod` once but stuck with `classmethod`, but I honestly don't recall why, so I won't push too hard on this one if you think `staticmethod` makes more sense...  5:)